### PR TITLE
cleanup values files

### PR DESCRIPTION
**Pull request by paulopontesm**

### CODE REVIEW

File: codacy/values.yaml [hunk 1]
```diff
@@ -339,71 +269,6 @@ codacy-ingress:
...
-        cpu: 1
-        memory: 3000000000 # this value needs to be set in bytes
-
-codacy-spa:
-  replicaCount: 1
-  fullnameOverride: codacy-spa
-  config:
-    codacy:
-      features:
-        passwordLogin: true
```
Comment (paulopontesm) [inline codacy/values.yaml:405]: this is already set in the chart itself:
https://github.com/codacy/codacy-spa/pull/179/files#diff-c7eecabc2da679d337f890d31b076513L4

File: codacy/values.yaml [hunk 2]
```diff
@@ -483,10 +353,3 @@ minio:
     purge: "false"
   defaultBucket:
     enabled: false
-
-ragnaros:
```
Comment (joaocosta-codacy) [inline codacy/values.yaml:487]: ragnaros is working

File: codacy/values.yaml [hunk 3]
```diff
@@ -194,130 +208,46 @@ global:
...
-    play:
-      configFile: 'enterprise-engine.conf'
-      logFile: 'logback.xml'
-  metrics:
-    serviceMonitor:
-      enabled: false
-    grafana_dashboards:
-      enabled: false
 
-codacy-tools:
```
Comment (joaocosta-codacy) [inline codacy/values.yaml:316]: codacy-tools is working

File: codacy/values.yaml [hunk 4]
```diff
@@ -194,130 +208,46 @@ global:
...
-  replicaCount: 1
-  fullnameOverride: codacy-tools
-  config:
-    environment: enterprise
```
Comment (joaocosta-codacy) [inline codacy/values.yaml:320]: codacy-tools is working

File: codacy/values.yaml [hunk 5]
```diff
@@ -483,10 +353,3 @@ minio:
     purge: "false"
   defaultBucket:
     enabled: false
-
-ragnaros:
-  fullnameOverride: codacy-ragnaros
-  config:
-    play:
-      configFile: enterprise.conf
-      logFile: application-logger.xml
```
Comment (joaocosta-codacy) [inline codacy/values.yaml:492]: ragnaros is working

File: codacy/values.yaml [hunk 6]
```diff
@@ -194,130 +208,46 @@ global:
...
-  config:
-    environment: enterprise
-    play:
-      configFile: 'enterprise-engine.conf'
-      logFile: 'logback.xml'
-  metrics:
-    serviceMonitor:
-      enabled: false
-    grafana_dashboards:
-      enabled: false
```
Comment (joaocosta-codacy) [inline codacy/values.yaml:314]: engine is working

File: codacy/values.yaml [hunk 7]
```diff
@@ -194,130 +208,46 @@ global:
...
     postgresqlDatabase: "activities"
-    postgresqlPassword: "PLEASE_CHANGE_ME"
+    postgresqlPassword: "codacy"
     service:
       port: 5432
-    fullnameOverride: codacy-activitiesdb
 
-remote-provider-service:
-  replicaCount: 1
-  fullnameOverride: codacy-remote-provider-service
```
Comment (joaocosta-codacy) [inline codacy/values.yaml:240]: remote-provider-service is working

File: codacy/values.yaml [hunk 8]
```diff
@@ -194,130 +208,46 @@ global:
...
       port: 5432
-    fullnameOverride: codacy-listenerdb
-
-core:
-  replicaCount: 1
-  fullnameOverride: codacy-core
-  config:
-    play:
-      configFile: "enterprise.conf"
-      logFile: logback.xml
```
Comment (joaocosta-codacy) [inline codacy/values.yaml:300]: core is working

**Pull request merged**